.github/workflows/ci.yml

```diff
@@ -17,25 +17,25 @@ jobs:
           # exit-zero treats all errors as warnings
           flake8 . --format="::warning file=%(path)s,line=%(row)d,col=%(col)d::%(text)s" --exit-zero --max-complexity=10
 
-  lint-black:
+  lint-pylint:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the repository at the current branch
         uses: actions/checkout@v2
       - name: Install dependencies
         uses: ./.github/actions/setup
-      - name: Lint with black
-        run: black --diff .
+      - name: Lint with pylint
+        run: pylint --msg-template='::({category}) file={path},line={line},col={column}::{msg}' *; if [ $(( $? & 2 )) -eq 2 ] || [ $(( $? & 1 )) -eq 1 ] ; then exit 1; else exit 0; fi
 
-  lint-pylint:
+  lint-autopep8:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the repository at the current branch
         uses: actions/checkout@v2
       - name: Install dependencies
         uses: ./.github/actions/setup
-      - name: Lint with pylint
-        run: pylint --msg-template='::warning file={path},line={line},col={column}::{msg}' */ || exit 0
+      - name: Lint with autopep8
+        run: autopep8 --recursive --diff --exclude ".venv/**" .
```

Comment:
Any reason for swapping the order of pylint & style checks?

Reply:
mainly grouping by tool functionality. flake8 and pylint at the top being pure linters and then autopep8 and isort being linters/formatters